monkey/monkey_island/cc/services/attack/technique_reports/T1110.py

```diff
@@ -35,11 +35,11 @@ def get_report_data():
                 result['successful_creds'].append(T1110.parse_creds(attempt))
 
         if succeeded:
-            data = T1110.get_message_and_status(T1110, ScanStatus.USED)
+            data = T1110.get_message_and_status(ScanStatus.USED)
```

Comment:
Like on T1075, set status according to succeeded and attempts, call get_message_and_status only once